Helper/Price.php

```diff
@@ -216,11 +208,60 @@ public function getProductFinalDisplayPrice(Product $product, Store $store)
      * @param Store $store
      * @return bool
      */
-    private function includeTaxes(Store $store)
+    public function includeTaxes(Store $store)
```

Comment:
I don't understand why this was made public if it's not used anywhere else in this PR. Maybe we can revert this?